matrix-generator.js

```diff
@@ -1,5 +1,21 @@
 function matrixGenerator(rows, columns) {
   // do work here
+  let results = [];
+  for (let i = 0; i < rows; i++) {
+    let rows = [];
+    for (let j = 0; j < columns; j++) {
+      let randomNum = Math.floor(Math.random() * 100) + 1;
+      rows.push(randomNum);
+      rows.sort(function(a, b) {
```

Comment:
This sorts everytime it adds to the inner array. If you place it outside of the inner for loop, it'll just do it once when the array is filled up.